features/steps/consistency.py

```diff
@@ -73,11 +73,11 @@ def step_impl(context, image, output_format):
     if output_format == 'COLOR_PROPHOTO_RGB':
         output_switch = '-color ProPhotoRGB'
     output_split = output_switch.split(' ')
-    # there's probably a more pythonic way to do this, with array flattening etc.
-    # just being expedient for the time being
-    args = [found_executable, '-tiff', output_split[0], image]
-    if len(output_split) == 2:
-        args = [found_executable, '-tiff', output_split[0], output_split[1], image]
+    if output_format != 'CROP':
+        output_split = output_split + ['-no-crop']
+    if output_format == 'CROP':
+        output_split = []
```

Comment:
You're right, it could be done with an else, but the array assignment isn't redundant, it's resetting `output_split` to a blank array.  Probably best to do something like:

    output_split = []
    if output_format != 'CROP':
      output_split = output_switch.split(' ') + ['-no-crop']

Then that way, on line 80, `output_split` is in the same format (ie, it's an array, just blank), so the code won't break.